src/app/component/comments/components/comment-pagination/comment-pagination.component.html

```diff
@@ -1,8 +1,9 @@
 <div class="pagination">
   <pagination-template #p="paginationApi"
                        [id]="config.id"
-                       [maxSize]="maxSize"
-                       (pageChange)="onPageChange($event)">
+                       [maxSize]="(p.getTotalItems())<=5 ? 5: (p.getCurrent()<=3 || p.getCurrent()>=(p.getLastPage()-2))?6:7"
```

Comment:
Would you think to move logical expressins to component.ts file?

Reply:
Calculation has been moved to .ts file.